azext_edge/edge/providers/orchestration/resources/dataflows.py

```diff
@@ -383,6 +389,119 @@ def __init__(self, cmd):
         self.instances = Instances(self.cmd)
         self.iotops_mgmt_client = self.instances.iotops_mgmt_client
         self.ops: "DataflowEndpointOperations" = self.iotops_mgmt_client.dataflow_endpoint
+        self.instances = Instances(self.cmd)
+
+    def create(
+        self,
+        name: str,
+        instance_name: str,
+        resource_group_name: str,
+        endpoint_type: DataflowEndpointType,
+        show_config: Optional[bool] = None,
+        **kwargs
```

Comment:
there is enormous reliance on kwargs here. the method looks to be designed to be flexible which is valuable but relying on so many kwargs especially when not explictly declared makes the operation harder to use and maintain ...its difficult to know what options are valid without digging into the implementation or documentation.

Reply:
also thinking of increased chance of silent bugs if the wrong or misspelled key is passed in, and linters or IDEs aren't as effective with autocomplete or static checks.

Reply:
Talked offline, created PBI https://msazure.visualstudio.com/One/_sprints/taskboard/IoT-UPX/One/Bromine/CY25Q2/2Wk/2Wk04%20(May%2018%20-%20May%2031)?workitem=33011831